### DISCUSSION TITLE
Land trade policy - multiple?

### DISCUSSION THREAD

**Discussion by catalin-duta** [Keriew/augustus#522] [Q&A]
![Screenshot_20211111-173012321](https://user-images.githubusercontent.com/93785920/141324791-d89ae4c5-98a2-4ccc-bee4-1d3726a58622.jpg)

In the above screenshot there are two things that seem contradictory to me:
- policy Consilium venditor terrae, so exports earn +20%
- capacity 12 units used, implying policy Consilium quantitas terrae

I should note that the Caravanserai was built, policy Venditor was set, then it was demolished and rebuilt. But the policy was never on Quantitas.
And as I understand, only 1 should be active at any given time.

**Comment by crudelios** [comment]
This is strange. Can you provide the savefile, please?

**Reply by catalin-duta** [reply]
Here it is. Using the latest build, forgot to mention.
[6Lugdunum8Ah21ltp.zip](https://github.com/Keriew/augustus/files/7521692/6Lugdunum8Ah21ltp.zip)

**Comment by crudelios** [comment]
I'm going to move this to an issue.
